rsts/community/troubleshoot.rst

```diff
@@ -85,13 +85,13 @@ Troubles with ``flyte sandbox`` log viewing
 
 .. note::
 
-   There is a ``skip`` button, which will take you straight to the logs without needing to log in.
+   There is a ``skip`` button, which will take you straight to the logs without logging in.
 
 
-Troubles with ``make start`` command in flytesnacks
+Troubles with ``make start`` command in Flytesnacks
```

Comment:
@SmritiSatyanV, can you remove this section? Can you add ` kubectl describe po <pod-name> -n flyte` command below "You can check on the pending pods and perform a detailed check as to why a pod is failing." in the first section?